core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/common/AbstractUserManagementErrorListener.java

```diff
@@ -332,4 +332,53 @@ public boolean onGetUserCountFailure(String errorCode, String errorMessage, Stri
         return true;
     }
 
+    @Override
+    public boolean onAddGroupFailure(String errorCode, String errorMessage, String groupName, List<String> userIDList,
+            UserStoreManager userStoreManager) throws UserStoreException {
+
+        return true;
+    }
+
+    @Override
+    public boolean onGetGroupFailure(String errorCode, String errorMessage, String groupID,
+            List<String> requiredAttributes, UserStoreManager userStoreManager) throws UserStoreException {
```

Comment:
Shall we make this requiredClaims following the same convention

Reply:
Updated the PR with the change.